devenv/lib/docker.py

```diff
@@ -91,7 +91,7 @@ def install_global() -> None:
 
     if shutil.which("docker", path=binroot) == f"{binroot}/docker":
         stdout = proc.run((f"{binroot}/docker", "--version"), stdout=True)
-        installed_version = stdout.strip().split()[2]
+        installed_version = stdout.strip().split()[2][:-1]
```

Comment:
Could we use regex instead? Or make clearer that the `[:-1]` is needed to remove the trailing comma (i.e. using `split(',')[0]`)?

Reply:
you can just `docker --version` and have a look yourself - yes, it's to remove the trailing comma